packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -474,7 +483,7 @@ const PricingOptionsFeatureList = forwardRef<HTMLDivElement, PricingOptionsFeatu
             reversedToggles
           >
             <ChevronDownIcon className={styles['PricingOptions__feature-list-accordion-chevron']} />
-            What&apos;s included
+            <FeatureListTitle />
```

Comment:
and finally


```suggestion
            {FeatureListTitle}
```

Makes no major difference though really, so feel free to disregard